packages/ckeditor5-emoji/docs/features/emoji.md

```diff
@@ -51,31 +63,47 @@ ClassicEditor
 	The emoji feature requires the {@link features/mentions mentions feature} to be installed to work properly.
 </info-box>
 
+The {@link module:emoji/emoji~Emoji emoji plugin} consists of two sub-plugins:
+
+* {@link module:emoji/emojimention~EmojiMention} &ndash; Adds support for triggering the emoji UI dropdown while writing the content when you type a pre-configured `:` marker followed by at least two letters of desired emoji name.
+* {@link module:emoji/emojipicker~EmojiPicker} &ndash; Registers the UI button component which opens a panel with a table of selectable emojis.
+
+You can freely choose how to load the feature: either as a whole (as shown in the example above), or select one of the sub-plugins if you only need a specific way of inserting emojis into the document.
+
 ## Configuration
 
-There are two configuration settings available.
+There is one configuration setting available for the emoji feature.
 
-* `marker` &ndash; the trigger character to evoke the emoji dropdown during edition. By default it is `:`.
 * `dropdownLimit` &ndash; the number of items to appear in the emoji dropdown.
 
-<!-- Please update the configuration snippet -->
-
 ```js
 ClassicEditor
 	.create( document.querySelector( '#editor' ), {
 		// ... Other configuration options ...
 		emoji: {
-			marker: :,
 			dropdownLimit: 5
 		}
 	} )
 	.then( /* ... */ )
 	.catch( /* ... */ );
 ```
 
+<info-box info>
+	The emoji feature already has a pre-configured `:` marker that opens a panel with a table of selectable emojis. If you are using {@link features/mentions mentions} or {@link features/merge-fields merge fields} features, they can also show UI panels by pressing a pre-configured key and it may be in conflict with the emoji feature.
+
+	To make all these features work fine together, make sure that {@link module:mention/mentionconfig~MentionFeed#marker mention's `marker`} and {@link module:merge-fields/mergefieldsconfig~MergeFieldsConfig#prefix merge field's `prefix`} configuration options are defined differently than `:`.
+</info-box>
+
+## Related features
+
+In addition to enabling emoji feature, you may want to check the following related features:
+
+* {@link features/special-characters Special characters} &ndash; Allows inserting **mathematical operators**, **currency symbols**, **punctuation**, **graphic symbols** (such as arrows or bullets), or Unicode letters typically not accessible from the keyboard (such as **umlauts** or **other diacritics**).
+* {@link features/mentions Mentions} &ndash; Brings support for smart autocompletion.
```

Comment:
I think that it does not make sense to put `Mention` here, as it is required by `Emoji` anyway.

Reply:
In this section we provide similar features and mention is very similar to emoji. Emoji underneath hack the mention plugin, but in my opinion it doesn't matter from the user's point of view. I would keep the link between mention and emoji features.